ra/ra.go

```diff
@@ -2754,3 +2755,14 @@ func validateContactsPresent(contacts []string, contactsPresent bool) error {
 func (ra *RegistrationAuthorityImpl) DrainFinalize() {
 	ra.finalizeWG.Wait()
 }
+
+// UnpauseAccount receives a validated account unpause request from the SFE and
+// instructs the SA to unpause that account. If the account cannot be unpaused,
+// an error is returned.
+func (ra *RegistrationAuthorityImpl) UnpauseAccount(ctx context.Context, request *rapb.UnpauseAccountRequest) (*emptypb.Empty, error) {
+	if core.IsAnyNilOrZero(request.RegistrationID) {
+		return nil, errIncompleteGRPCResponse
+	}
+
+	return nil, errors.New("unimplemented")
```

Comment:
```suggestion
	return nil, status.Errorf(codes.Unimplemented, "method UnpauseAccount not implemented")
```

where `status` is google.golang.org/grpc/status